### PR TITLE
XIVY-14735 write an error-log file

### PR DESCRIPTION
**Pull request by ivy-rew**
- do not show error details to end-users

slim-error-log

### CODE REVIEW

File: src/app/Application.php [hunk 1]
```diff
@@ -78,4 +81,12 @@ private static function createContainer($config): Container
 
         return $container;
     }
+
+    public function setupLogging(App $app)
+    {
+        $app->addRoutingMiddleware();
+        $logger = new Logger('error');
+        $logger->pushHandler(new RotatingFileHandler('../../logs/slim-error.log', 30));
+        $app->addErrorMiddleware(false, true, true, $logger);
```
Comment (ivy-rew) [inline src/app/Application.php:90]: false = display no errors to end user

File: src/app/Application.php [hunk 2]
```diff
@@ -78,4 +81,12 @@ private static function createContainer($config): Container
 
         return $container;
     }
+
+    public function setupLogging(App $app)
+    {
+        $app->addRoutingMiddleware();
+        $logger = new Logger('error');
+        $logger->pushHandler(new RotatingFileHandler('../../logs/slim-error.log', 30));
```
Comment (ivy-rew) [inline src/app/Application.php:89]: 30 = retention days

Comment (ivy-rew) [inline src/app/Application.php:89]: ../../ should reference to deployment root :crossed_fingers: 

axonivy-website-dirs

**Review by alexsuter — APPROVED** [review]
:+1:

**Pull request merged**